src/components/Note.js

```diff
@@ -68,7 +68,7 @@ const Note = (props) => {
   }, [type, expired, expiresAt])
 
   const onLinkClick = (e) => {
-    if (expired) e.preventDefault();
+    // if (expired) e.preventDefault();
```

Comment:
Do not leave commented code. If you want to make a note accessible from the Archive, remove onLickClick at all. 

Btw, I dunno maybe you already implemented it, but I would like  archived notes to be distinguishable from the regular ones when they are displayed. Maybe by bg color of input fields or something else. Think about it